.github/workflows/pr-testing-with-test-project.yml

```
@@ -9,7 +9,7 @@ jobs:
   changes:
     runs-on: ubuntu-latest
     outputs:
-      should_test: ${{ steps.filter.outputs.modified_files !='[]' }}
+      should_test: ${{ steps.filter.outputs.modified_files }}
```

actionlint (1.7.4)

property "filter" is not defined in object type {}

(expression)